Resources/public/sonata-page.back.css

```diff
@@ -403,6 +403,15 @@
   margin: 0 !important;
   font-size: 14px !important; }
 
+/*# sourceMappingURL=composer.css.map */
```

Comment:
you need to remove the mapping information to avoid error when file are concatenated.